src/client/components/Season.jsx

```diff
@@ -14,10 +14,10 @@ export const Season = () => {
             
             {/*dropdown needs styled*/}
             <select className="select-round-dropdown" name="rounds">
-                <option value="1">{`Mario Circuit`}</option>
-                <option value="2">{`Coconut Mall`}</option>
-                <option value="3">{`DK Summit`}</option>
-                <option value="4">{`Wario's Gold Mine`}</option>
+                <option className="dropdown-option" value="1">{`Mario Circuit`}</option>
```

Comment:
Set these as constants in some config' file somewhere?

Reply:
these are placeholders, they will be changed as we continue to merge frontend and backends together